client/packages/requisitions/src/RequestRequisition/ListView/ListView.tsx

```diff
@@ -41,26 +42,38 @@ export const RequestRequisitionListView: FC = () => {
   const { data, isError, isLoading } = useRequest.document.list();
   const pagination = { page, first, offset };
   useDisableRequestRows(data?.nodes);
+  const { requireSupplierAuthorisation } = useRequest.utils.preferences();
 
-  const columns = useColumns<RequestRowFragment>(
+  const columnDefinitions: ColumnDescription<RequestRowFragment>[] = [
+    [getNameAndColorColumn(), { setter: onUpdate }],
+    {
+      key: 'requisitionNumber',
+      label: 'label.number',
+      width: 100,
+    },
+    'createdDatetime',
     [
-      [getNameAndColorColumn(), { setter: onUpdate }],
+      'status',
       {
-        key: 'requisitionNumber',
-        label: 'label.number',
-        width: 100,
+        formatter: currentStatus =>
+          getRequisitionTranslator(t)(currentStatus as RequisitionNodeStatus),
       },
-      'createdDatetime',
-      [
-        'status',
-        {
-          formatter: currentStatus =>
-            getRequisitionTranslator(t)(currentStatus as RequisitionNodeStatus),
-        },
-      ],
-      ['comment', { width: '100%' }],
-      'selection',
     ],
+    ['comment', { width: '100%' }],
+  ];
+
+  if (requireSupplierAuthorisation) {
```

Comment:
approvalStatus is enum now, once the initial PR is merged would be great to do translations for it